applications/nrf_desktop/src/modules/usb_state.c

```diff
@@ -86,7 +86,17 @@ static struct usb_hid_device usb_hid_device[] = {
 static struct usb_hid_device usb_hid_device[CONFIG_USB_HID_DEVICE_COUNT];
 #endif
 
-BUILD_ASSERT(DT_PROP(DT_NODELABEL(usbd), num_in_endpoints) > ARRAY_SIZE(usb_hid_device),
+#if IS_ENABLED(CONFIG_DESKTOP_USB_STACK_NEXT) && DT_NODE_EXISTS(DT_NODELABEL(usbhs))
```

Comment:
Can we make it `#if CONFIG_DESKTOP_USB_STACK_NEXT`? To improve consistency, We do not use `IS_ENABLED` in similar use-cases.